api/references/when-clause-contexts.md

```diff
@@ -257,10 +257,10 @@ Context name | True when
 
 You can have a when clause that checks if a specific [View](/api/ux-guidelines/views) is visible or focused.
 
-Context name | True when
------------- | ------------
+Context name  | True when
```

Comment:
```suggestion
Context name | True when
```

Reply:
The added space makes the `|` on table title row aligned with that same characters on the separator row (`------------- | ----------`) and on the row for `` `focusedView` ``.

Reply:
Okay. thanks